controller/src/main/java/io/pravega/controller/server/rpc/auth/PravegaInterceptor.java

```diff
@@ -69,6 +70,8 @@
                 context = context.withValue(AUTH_CONTEXT_PARAMS, paramMap);
                 context = context.withValue(INTERCEPTOR_OBJECT, this);
             }
+        } else {
+            throw new RuntimeException(new AuthenticationException("Handler not specified"));
```

Comment:
This scenario seems like a user error, i.e. it is expected.   Does throwing `RuntimeException` here produce a stack trace in the server log, and if so can that be avoided?   Seems a better fix would be:
```
} else {
    call.close(Status.fromCode(Status.Code.UNAUTHENTICATED), headers);
}
```

Reply:
Fixed. Thanks.